src/jit/target.h

```diff
@@ -678,6 +678,12 @@ typedef unsigned short regPairNoSmall; // arm: need 12 bits
 
   #define RBM_ARG_REGS            (RBM_ARG_0|RBM_ARG_1)
 
+  // The registers trashed by profiler enter/leave/tailcall hook
+  // See vm\i386\asmhelpers.asm for more details.
+  #define RBM_PROFILER_ENTER_TRASH     RBM_NONE
+  #define RBM_PROFILER_LEAVE_TRASH     RBM_NONE
+  #define RBM_PROFILER_TAILCALL_TRASH  (RBM_ALLINT & ~RBM_ARG_REGS)
```

Comment:
Whenever we implemented ELT profiler hooks, we run 250 odd desktop profiler tests to ensure that there are no issues.  Please run the same before check-in, if you have not already run.
Another test that I do is ngen mscorlib successfully with complus_JitELTHookEnabled=1.